changelog/v0.31.0/hashable-input-resources.yaml

```diff
@@ -0,0 +1,7 @@
+changelog:
+  - type: BREAKING_CHANGE
```

Comment:
If this is breaking, how should consumers of this library upgrade? Can we call out in the PR how a developer in gloo for example would bring in this change?

Reply:
no longer a breaking change